mlflow/protos/model_registry.proto

```diff
@@ -529,8 +529,8 @@ message SearchModelVersions {
   // with string values wrapped in single quotes.
   optional string filter = 1;
 
-  // Maximum number of models desired. Max threshold is 200K.
-  optional int64 max_results = 2 [default = 200000];
```

Comment:
@serena-ruan @harupy Can we revert this change to avoid making backwards-incompatible changes to the MLflow REST API? Can we instead add the following sentence to the docstring: "Backends may choose a lower default value and maximum threshold." ?

Reply:
@dbczumar Reverted the change and added the sentence 👍